### DISCUSSION TITLE
Finding your Flight in Mission Editor

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#1779] [Q&A] [answered]
Hi, when I open liberation_nextturn in the ME, I see dozens of blue plane icons and flight lines all over the place. Is there a button or command I can use in ME to get the player mission/plane that I have selected from Liberation to automatically show up (in white) so I can do my edits to my flight like change the loadout, skin, etc? Otherwise, I have to click on many of those blue lines and plane icons to figure out which one is my mission/plane.    

UPDATE:  'C' key in ME centers player's plane icon in screen and highlights it in white.

**Comment by GrEaSeLiTeNiN** [comment, marked as answer]
'C' key in ME centers player's plane icon in screen and highlights it in white. If that doesn't work, seems it's because "Player" is showing up as "Client". Not sure why that is.

**Reply by Starfire13** [reply]
There is also an icon of a human head you can click on that selects the player aircraft.

**Reply by GrEaSeLiTeNiN** [reply]
Is there such an icon in ME? I do not see it...

**Reply by Starfire13** [reply]
![Capture](https://user-images.githubusercontent.com/72491792/143530983-317c282d-fe0f-4595-ada7-92b9297a91ae.JPG)

**Reply by GrEaSeLiTeNiN** [reply]
Ok yes, thanks
